certbot/storage.py

```diff
@@ -204,7 +207,7 @@ class RenewableCert(object):  # pylint: disable=too-many-instance-attributes
         renewal configuration file and/or systemwide defaults.
 
     """
-    def __init__(self, config_filename, cli_config):
+    def __init__(self, config_filename, cli_config, update_symlinks=False):
```

Comment:
Being required to call a method immediately after instantiating an object seems like a more complicated API. The idea of putting it in the constructor is that sometimes you're asked to construct an object from a renewal file that is currently not in a valid state, and to properly construct it you'll have to go in and fix that state.